content/events/2020/07/2020-07-28-2020-government-ux-summit.md

```diff
@@ -95,7 +95,7 @@ This presentation proposes a useful method to track standards and UX requirement
 Designing and developing digital products that are accessible to all doesn’t just make good business sense — it’s also the law. Yet it’s often an overlooked part of the product development lifecycle. The best and most cost-effective way to ensure your product is accessible is to start with accessibility. In this session, participants will learn how to incorporate accessibility and inclusive design at each stage of the product lifecycle. From product requirements through user research, content strategy and copywriting, design, front-end development, and QA, consider accessibility using these effective guidelines. 
    
 - [Watch the video](https://www.youtube.com/watch?v=ZFdLZlAD5q0)
-- [Read the transcript](https://digital.gov/pdf/2020-07-28-UX-Summit-Captioning-Tues-PM.pdf) (PDF, 119 KB, 13 pages) 
+- [Read the transcript](https://s3.amazonaws.com/digitalgov/static/2020-07-28-UX-Summit-Captioning-Tues-PM.pdf) (PDF, 119 KB, 13 pages)
```

Comment:
Shortcode?

Reply:
@mejiaj The `static-asset` shortcode is not rendered when nested within the `accordion` shortcode, it's the only file that has this problem, hence why I included the full path to allow the `li` to render as expected.

The screenshot shows how the markup is rendered when the `static-asset` shortcode is used:

<img width="939" alt="Screen Shot 2022-11-25 at 11 48 29 AM" src="https://user-images.githubusercontent.com/104778659/204030282-eaef6fa4-8e7e-40a7-9ad8-7577bb4c8b19.png">

Reply:
Okay, good find. This is an important note and will have to workaround this limitation. Can you create an issue for this?